lang/syn/src/codegen/program/accounts.rs

```diff
@@ -3,7 +3,7 @@ use heck::SnakeCase;
 use quote::quote;
 
 pub fn generate(program: &Program) -> proc_macro2::TokenStream {
-    let mut accounts = std::collections::HashSet::new();
+    let mut accounts = std::collections::BTreeMap::new();
```

Comment:
Why not HashMap?

Reply:
> Why not HashMap?

I don't remember I did the PR a long time ago

Reply:
But yeah it's better you right, we want to use this to cache values

Reply:
It doesn't matter here actually, this is just macro generation code so it doesn't run on-chain.